### DISCUSSION TITLE
How to create a simple point?

### DISCUSSION THREAD

**Discussion by radar4ick** [microsoft/maker.js#616] [Q&A] [answered]
POINT
  5
30F
  8
NOTCH$CCW$L
 10
1871.0
 20
6.8011667117243633
 30
0.0
  0

HI! I need the simple point information to be in the exported file. How can I do this? I didn't find the solution I needed in the documentation

**Comment by danmarshall** [comment, marked as answer]
Maker.js ought to have a better way of handling DXF entities. Until then, you might be able to inject into the entities section. Here is example DXF for a drawing with only a circle:

```dxf
0
SECTION
2
HEADER
9
$INSUNITS
70
4
0
ENDSEC
0
SECTION
2
TABLES
0
TABLE
2
LTYPE
0
LTYPE
72
65
70
64
2
CONTINUOUS
3
______
73
0
40
0
0
ENDTAB
0
TABLE
2
LAYER
0
ENDTAB
0
ENDSEC
0
SECTION
2
ENTITIES
0
CIRCLE
8
0
10
0
20
0
40
50
0
ENDSEC
0
EOF
```
I believe you can add your points before the end of the final section. I haven't tried this, but here's a function written by ChatGPT:
```js
function injectPoint(dxfData, pointData) {
  // Find the index of the last line of the ENTITIES section before it ends
  const endEntitiesIndex = dxfData.lastIndexOf('0\nENDSEC\n', dxfData.indexOf('0\nSECTION\n2\nENTITIES'));
  if (endEntitiesIndex === -1) {
    return "Invalid DXF data"; // or handle error more appropriately
  }

  // Insert the point data before the end of the ENTITIES section
  return dxfData.substring(0, endEntitiesIndex) + pointData + dxfData.substring(endEntitiesIndex);
}
```
